tools/build_langserver/lsp/completion.go

```diff
@@ -111,7 +111,7 @@ func (h *Handler) completeIdent(doc *doc, s string, line, col int) (*lsp.Complet
 	list := &lsp.CompletionList{}
 	for name, f := range h.builtins {
 		if strings.HasPrefix(name, s) {
-			item := completionItem(name, s, line, col)
+			item := completionItem(name, "", line, col)
```

Comment:
If we're completing "check_" to "check_config", name would be "check_config", and s would have been "check_". We would then produce a replacement from "check_" to "config", rather than to the full "check_config" because we strip this prefix. I think this prefix is for when we have something like //foo: -> foo resulting in `//foo:foo`,